packages/@react-spectrum/s2/src/ContextualHelp.tsx

```diff
@@ -40,9 +40,8 @@ export interface ContextualHelpProps extends
 }
 
 const popover = style({
-  fontFamily: 'sans',
-  minWidth: 268,
-  width: 268,
+  minWidth: 'calc(268px - 2 * var(--s2-container-border-width))',
+  width: 'calc(268px - 2 * var(--s2-container-border-width))',
```

Comment:
these won't be rem anymore, so the size may change, should we do the calculation and use rems here do you think?

should just be `calc(268px / 16 - (2 * var(--s2-container-border-width))` since the border isn't rem

also, could you leave it as it was but change the `boxSizing` to `borderBox`? that should have the same effect

Reply:
gah, silly inner vs outer, you are correct